docs/FAQ.md

```diff
@@ -0,0 +1,59 @@
+# FAQ
+
+Frequently asked questions, please read these before creating a new issue.
+
+## Can I use Spegel in production?
+
+We have been running Spegel in multiple production Kubernetes clusters at Xenit since the first release without any issues. The great thing is that pulling images would not stop working if you for some reason would find an issue with Spegel.
+A fallback to the original registry will always occur if Spegel can't be reached or serve the requested image.
+
+## How do I know that Spegel is working? 
+
+Spegel is meant to be a painless experience to install, meaning that it may be difficult initially to know if things are working or not. Simply put a good indicator that things are working is if all Spegel pods have started and are in a ready state.
+Spegel does a couple of checks on startup to verify that any required configuration is correct, if it is not it will exit with an error. While it runs it will log all received requests, both those it mirrors and it serves.
+
+An incoming request to Spegel that is mirrored will receive the following log.
+
+```
+{"level":"info","ts":1692304805.9038486,"caller":"gin@v0.0.9/logger.go:53","msg":"","path":"/v2/library/nginx/blobs/sha256:1cb127bd932119089b5ffb612ffa84537ddd1318e6784f2fce80916bbb8bd166","status":200,"method":"GET","latency":0.005075836,"ip":"172.18.0.5","handler":"mirror"}
+```
+
+While the Spegel instance on the other end will log.
+
+```
+{"level":"info","ts":1692304805.9035861,"caller":"gin@v0.0.9/logger.go:53","msg":"","path":"/v2/library/nginx/blobs/sha256:1cb127bd932119089b5ffb612ffa84537ddd1318e6784f2fce80916bbb8bd166","status":200,"method":"GET","latency":0.003644997,"ip":"172.18.0.5","handler":"blob"}
+```
+
+## Why am I not able to pull the new version of my tagged image?
```

Comment:
@dimm0 @pixeye33

I had some discussions with a friend about some alternative solutions to deal with updating tags, and we came up with an alternative solution. It way be of interest for you to checkout.